image_assets/models.py

```diff
@@ -132,7 +132,7 @@ def validate_aspect(self, file: Image.Image) -> List[str]:
             if image_aspect != self.aspect:
                 msg = _('Image aspect must be %s')
                 return [msg % self.aspect]
-        elif round(delta / self.accuracy) > 1:
+        elif round(delta / self.accuracy, 4) > 1:
```

Comment:
Why not `delta > self.accuracy`?

Reply:
Because here is the calculation of floating point numbers: 
```
>>> accuracy = 0.1
>>> delta = 2.1 - 2.0
>>> delta
0.10000000000000009

>>> delta > accuracy
True  - wrong

>>> round( delta / accuracy, 4 ) > 1
False - correctly
```

could be done like this: 
```
>>> round( delta, 4 ) > accuracy
False - correctly
```
but i just updated the existing solution